pydash/pydash_app/dashboard/fetch.py

```diff
@@ -0,0 +1,66 @@
+import requests
+import jwt
+import json
+
+
+def get_details(dashboard_url):
+    """
+    Get details from a deployed flask-monitoring-dashboard
+    :param dashboard_url: The base URL for the deployed dashboard, without trailing slash
+    :return: A dict containing details from the dashboard, or None if the request was unsuccessful
+    """
+    endpoint = 'get_json_details'
```

Comment:
I'd love to move this to a config file, but I'm not sure if adding the endpoints to the `Config` class and using them directly would be a good idea.

Reply:
Clearly, the endpoint name is used in multiple functions. What you could do is create a simple: 

```python
def _endpoint_name(): 
    'get_json_details'
```

function, which can then easily later be adapted to read the endpoint name from some dynamic source.
I completely agree that using the `Config` class (which is part of Flask?) will complect the application with the web-interface, and probably making this configurable is too complex a solution _(for now, pre-MVP)_ either way.

Reply:
Do you mean an inner function?

Reply:
Yes, I mean a function that is only used inside this module.